blackbox-test/build-examples.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# This script is used for building all the example programs, when run from TravisCI.
+
+BBT_LC=$(pwd)/target/lc-blackbox
+EXAMPLES="$(pwd)/examples/*"
+
+set -e
+
+for dir in $EXAMPLES; do
+  if [ -d $dir ]; then
+    echo "*** Building example: ${dir} ***"
+    cd $dir
+    $BBT_LC ci
```

Comment:
gah, sorry @joeygibson this actually doesn't work because the publish task is designed to not work for the examples, probably just need to do:

```
$BBT_LC bootstrap
$BBT_LC test
$BBT_LC blackbox-test
```